curriculum/challenges/english/10-coding-interview-prep/rosetta-code/count-the-coins.md

```diff
@@ -30,7 +30,7 @@ There are four types of common coins in [US](https://en.wikipedia.org/wiki/Unite
 
 # --instructions--
 
-Implement a function to determine how many ways there are to make change for a dollar using these common coins (1 dollar = 100 cents)
+Implement a function to determine how many ways there are to make change for a `cents` cents using these common coins.
```

Comment:
"`cents` cents" sounds a little strange, but it's accurate I suppose. What about this:
```suggestion
Implement a function to determine how many ways there are to make change for a given input, `cents`, that represents an amount of US pennies, using these common coins.
```

I suppose if we went with dollars, we could test `1` dollar and `2` dollars. I think I like cents better so we don't exceed one dollar in our tests, which is what the original challenge asks for.

Reply:
@moT01 Is the last comma needed?  There seem to be too many commas in the sentence.